examples/QESR_rescale.py

```diff
@@ -0,0 +1,23 @@
+import pyspecdata as psd
+from pyspecProcScripts import QESR_scalefactor
+import matplotlib.pyplot as plt
+
```

Comment:
this thing is missing a docstring.  In the docstring, can you give an example of the pyspecdata config parameters that are needed for it to run correctly? (give in proper rst, make sure that sphinx gallery page builds)